index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Mobile First Project</title>
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <a href="#" class="logo">My Logo</a>
+      <nav class="hamburger">
+        <span class="navbar"></span>
+        <span class="navbar"></span>
+        <span class="navbar"></span>
+      </nav>
+    </header>
+
+    <section class="main-container">
+      <div class="title">
+        <h1>This is Mert Glad to see you here!</h1>
+      </div>
+      <dev class="intro">
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+        <p class="connect">LET'S CONNECT</p>
+
+        <div class="social-media">
+          <ul class="social-media-icons">
+            <li>
+              <img class="icons" src="Images/twitter.png" alt="twitter logo" />
+            </li>
+            <li>
+              <img
+                class="icons"
+                src="Images/Linkedin.png"
+                alt="linkedin logo"
+              />
+            </li>
+            <li>
+              <img class="icons" src="Images/medium.png" alt="medium logo" />
+            </li>
+            <li>
+              <img class="icons" src="Images/github.png" alt="github logo" />
+            </li>
+            <li>
+              <img
+                class="icons"
+                src="Images/angellist.png"
+                alt="angellist logo"
+              />
+            </li>
+          </ul>
```

Comment:
- Great job rendering your list of icons with a `ul` tag to group them, and `li` per icon 💪🏽 
- I'd like to kindly suggest that each icon should have a link (`a` tag). Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

We can do something like this 
```
<li>
    <a href ="http://yourgithubhandle.com">
        <img
           class="icons"
            src="Images/github.png"
             alt="angellist logo"
          />
      </a>
  </li>
```

Why do't we refactor each of your icon to look something like the code snippet i have added above? 🤗

Reply:
Thanks for the suggestion. I updated the icons. However, I still have dots next to my icons and I can't get rid of them. What can I do in order to do it?